src/en/models.md

```diff
@@ -92,13 +92,30 @@ Common model management tasks are summarized below.
    
    Use the `juju switch` command to go from one model to another:
    
-   `juju switch [<controller>|<model>|<controller>:<model>]`
+   `juju switch [<controller or model>|<:model>|<controller>:<model>|<controller:>]`
    
    Running the command with no arguments will return the currently active 
    controller and model:
      
          juju switch
    
+   To change to a model:
+   
+   <!-- WARNING: EXTRA SPACES AT EOL -->
+   
+   `juju switch foo`  
+   Selects the last used model in controller 'foo' (if the latter exists),
```

Comment:
does this mean if you have a controller foo, and a model bar:foo then:

juju switch bar
juju switch foo

will take you to back to the foo container?

Reply:
If controller foo exists, the command `juju switch foo` will, irrespective of previous commands ("where you are"), always take you to the last used model in controller foo.

Reply:
how confusing